### ISSUE TITLE
Resource file not found

### ISSUE THREAD

**Issue by nmhild** [viniciusgerevini/godot-aseprite-wizard#139]
Was using this plugin in a new project and the player character animations imported without issue. After that, trying to import any other Aseprite file from any other file location gives me a "Resource file not found: /ase_NameOfFile.png (expected type: Image) error. This results in the animations and frames being imported, but the texture doesn't generate for them. No edits were made to any of the scripts present in the addon. Disabling and reenabling the addon did not fix the problem, nor did deleting it and reinstalling.

This is in the Mono version of Godot v4.2 stable.
![Godot_v4 2-stable_mono_win64_LA1imaBSxM](https://github.com/viniciusgerevini/godot-aseprite-wizard/assets/49663878/4bfee992-37b0-4047-af6c-c923db54f8fc)
![Godot_v4 2-stable_mono_win64_SGbA9ntEE7](https://github.com/viniciusgerevini/godot-aseprite-wizard/assets/49663878/91b5347f-b708-4988-ba58-522b49d475e5)
![Godot_v4 2-stable_mono_win64_Mes3DUQITB](https://github.com/viniciusgerevini/godot-aseprite-wizard/assets/49663878/372e5006-2dac-4525-a38e-d71a969cae45)

**Comment by viniciusgerevini** [comment]
Interesting. That path is as if it's trying to read the file from the root. I haven´t seen this happening before. Which operation system are you using?
I will try to reproduce it. Also, if you are comfortable sharing your new project or any reproducible project (with .godot / .import folders) it might make it easier to identify the issue.

**Comment by nmhild** [comment]
I'm using Windows 10 as my operating system. For whatever it's worth, the next day I tried again and each of the files that failed before, worked. As stated, I tried restarting Godot to fix the issue (a few times), so the only real difference is that I restarted my system.

If there's a way I can share my project with you privately, I'd be happy to (I love your plugin and would be happy to help), I just would rather not share it publicly.
